<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordManager</class>
 <widget class="QMainWindow" name="PasswordManager">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <item>
       <widget class="QPushButton" name="addEntryButton">
        <property name="text">
         <string>‚ûï Add Entry</string>
        </property>
        <property name="styleSheet">
         <string>
          QPushButton {
           background-color: #2D89EF;
           color: white;
           font-size: 14px;
           font-weight: bold;
           border-radius: 5px;
           padding: 5px;
          }
          QPushButton:hover {
           background-color: #1A5DBE;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBar">
        <property name="placeholderText">
         <string>üîç Search...</string>
        </property>
        <property name="styleSheet">
         <string>
          QLineEdit {
           border: 2px solid #A9A9A9;
           border-radius: 5px;
           padding: 5px;
           font-size: 14px;
          }
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="passwordTable">
      <property name="columnCount">
       <number>4</number>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>Website</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Password</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notes</string>
       </property>
      </column>
      <property name="styleSheet">
       <string>
        QTableWidget {
         background-color: white;
         border-radius: 5px;
         font-size: 12px;
        }
        QHeaderView::section {
         background-color: #2D89EF;
         color: white;
         font-weight: bold;
         padding: 5px;
         font-size: 13px;
        }
        QTableWidget::item {
         padding: 5px;
        }
        QTableWidget::item:selected {
         background-color: #ADD8E6;
        }
       </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
